cli/src/pcluster/config/cluster_config.py

```diff
@@ -2046,6 +2046,21 @@ def instance_types(self) -> List[str]:
             for flexible_instance_type in self.instances  # pylint: disable=not-an-iterable
         ]
 
+    def _min_schedulable_memory_and_instance_type(self):
+        instances_and_memory = {t: info.ec2memory_size_in_mib() for t, info in self.instance_type_info_map.items()}
+        smallest_type = min(instances_and_memory, key=instances_and_memory.get)
+        return smallest_type, instances_and_memory[smallest_type]
+
+    def _register_validators(self, context: ValidatorContext = None):
+        super()._register_validators(context)
```

Comment:
Please double-check if this call to the super's `_register_validators()` is necessary.